components/konflux-rbac/policies/bootstrap-tenant-namespace/bootstrap-tenant-namespace-clusterpolicy.yaml

```diff
@@ -34,3 +34,151 @@ spec:
           kind: ServiceAccount
           namespace: '{{request.object.metadata.name}}'
           name: appstudio-pipeline
+  - name: create-allow-from-console-namespaces-networkpolicy
+    match:
+      any:
+      - resources:
+          kinds:
+          - Namespace
+          selector:
+            matchLabels:
+              konflux-ci.dev/type: tenant
+    generate:
+      generateExisting: true
+      synchronize: false
```

Comment:
why synchronize is false? don't we want to update the NetworkPolicy if the Kyverno ClusterPolicy is changed?

Reply:
for the current implementation it could collide with KubeSaw's managed ones. We can modify the rules to take in consideration KubeSaw's labels or just wait for KubeSaw to be removed. I'd propose to do that in a dedicated PR if/when we need to update the resources.